static/js/lib/embed_test.js

```diff
@@ -48,4 +58,59 @@ describe("embed utils", () => {
     assert.equal(link.getAttribute("href"), url)
     assert.equal(link.getAttribute("class"), "embedly-card")
   })
+
+  describe("initialization", () => {
+    let embedlyStub
+
+    beforeEach(() => {
+      document.head.innerHTML = ""
+      document.body.innerHTML = ""
+
+      // this expects at least one script element to already exist
+      embedlyStub = sandbox.stub()
+      global.embedly = embedlyStub
+
+      const script = document.createElement("script")
+      document.head.appendChild(script)
+    })
+
+    it("loads embedly", () => {
+      loadEmbedlyPlatform()
+      assert.equal(document.querySelectorAll("script").length, 2)
+      const firstScript = document.querySelectorAll("script")[0]
+      assert.equal(
+        firstScript.getAttribute("src"),
+        "http://cdn.embedly.com/widgets/platform.js"
+      )
+    })
+    ;[true, false].forEach(hidesTitle => {
```

Comment:
nit: can you put a `//` line above tests like this, so there's a blank line b/tw tests?